Framework/DataHandling/test/LoadDetectorInfoTest.h

```diff
@@ -371,42 +371,32 @@ class LoadDetectorInfoTest : public CxxTest::TestSuite {
         TS_ASSERT(par);
         if (!par)
           return;
-        if (singleWallPressure) {
-          TS_ASSERT_DELTA(par->value<double>(),
-                          boost::lexical_cast<double>(pressure[j]), 1.e-3);
-        } else {
-          TS_ASSERT_EQUALS(par->asString(), castaround(pressure[j]));
-        }
+        TS_ASSERT_DELTA(par->value<double>(),
+                        boost::lexical_cast<double>(pressure[j]), 1.e-3);
```

Comment:
In the future you may want to use the std and less verbose[`std::stod`](http://en.cppreference.com/w/cpp/string/basic_string/stof).